docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -20,6 +20,28 @@ While this document is not normally applicable to normal silverstripe contributo
 it is still useful to have it available in a public location so that these users
 are aware of these processes.
 
+## Module releases
+
+Occasionally a fix to an individual module warrants a patch release outside of the standard quarterly release cycle. Rather than generating a full new recipe release, the following process should be followed to perform a patch release for an individual module:
+
+1. Check out the module locally
+1. Determine the new module version to be released (e.g. current version is 4.6.1, therefore new version will be 4.6.2)
```

Comment:
Should this be "current version is 4.6.0" and "new version will be 4.6.1" to align with the rest of the change? Or use "4.6.2 as 4.6.0" further down.

Reply:
Nice spotting 👍